source/analysis/AM1DDerivativeAB.cpp

```diff
@@ -301,15 +296,15 @@ bool AM1DDerivativeAB::values(const AMnDIndex &indexStart, const AMnDIndex &inde
 				badIndices.append(totalSize-1);
 
 			// Compute all the values
-			outputValues[0] = (data.at(1)-data.at(0))/(axis.at(1)-axis.at(0));
-			outputValues[totalSize-1] = (data.at(totalSize-1)-data.at(totalSize-2))/(axis.at(totalSize-1)-axis.at(totalSize-2));
+			cachedOutput_[0] = (data.at(1)-data.at(0))/(axis.at(1)-axis.at(0));
```

Comment:
You are correct, however, this probably doesn't affect anything because that value is not used and then is overwritten by the bad indices check.  I agree that it should be checked and fixed though.  Good catch.